cmd/config.go

```diff
@@ -215,10 +303,26 @@ func writeFile(filename string, data []byte, perm os.FileMode) error {
 	return err
 }
 
+func readKeyFromFile(filepath string) (*keymgmt.NACLKey, error) {
+	keyBytes, err := ioutil.ReadFile(filepath)
+	if err != nil {
+		panic(err)
+	}
+
+	keyStruct, err := keymgmt.UnmarshalNACLKey(keyBytes)
```

Comment:
You can shorten this function by replacing the last 5 lines with `return keymgmt.UnmarshalNACLKey(keyBytes)`